corehq/apps/app_manager/management/commands/import_app.py

```diff
@@ -78,4 +78,5 @@ def handle(self, domain, app_id, **options):
         app_source = resp.json()
         if not name:
             name = app_source['name']
-        import_app(app_source, target_domain, {'name': name})
+        app = import_app(app_source, target_domain, {'name': name})
+        return "Created app '{}' at /a/{}/apps/view/{}/".format(app.name, app.domain, app.id)
```

Comment:
Just looked at the Django source for `BaseCommand` and apparently if `handle` returns something, it writes it to `sys.stdout`.  Learned something new today.